src/Ledger/Ratify.lagda

```diff
@@ -263,39 +266,62 @@ actualVotes Γ pparams cc ga votes
   roleVotes r = mapKeys (uncurry credVoter) (filter (λ (x , _) → r ≡ proj₁ x) votes)
 
   activeDReps = dom (filter (λ (_ , e) → currentEpoch ≤ e) dreps)
-  spos = filterˢ IsSPO (dom (stakeDistr stakeDistrs))
+  spos  = filterˢ IsSPO (dom (stakeDistr stakeDistrs))
+\end{code}
+\begin{code}
 
   getCCHotCred : Credential × Epoch → Maybe Credential
   getCCHotCred (c , e) = case ¿ currentEpoch ≤ e ¿ᵇ , lookupᵐ? ccHotKeys c of
 \end{code}
 \begin{code}[hide]
-    λ where
+      λ where
 \end{code}
 \begin{code}
-      (true , just (just c'))  → just c'
-      _                        → nothing -- expired, no hot key or resigned
+        (true , just (just c'))  → just c'
+        _                        → nothing -- expired, no hot key or resigned
+
+  SPODefaultVote : GovAction → VDeleg → Vote
+  SPODefaultVote ga (credVoter SPO (KeyHashObj kh)) = case lookupᵐ? pools kh of
+\end{code}
+\begin{code}[hide]
+      λ where
+\end{code}
+\begin{code}
+        nothing → Vote.no
+        (just  p) → case lookupᵐ? delegatees (PoolParams.rewardAddr p) , ga of
+\end{code}
+\begin{code}[hide]
+               λ where
+\end{code}
+\begin{code}
+               (just noConfidenceRep , NoConfidence)  → Vote.yes
+               (_ , TriggerHF _)                      → Vote.no
+               (just abstainRep , _)                  → Vote.abstain
+               _                                      → Vote.no
```

Comment:
I think it would help readability if the `TriggerHF` line would be at the top and the commas would be aligned.